src/x509/clu_request_setup.c

```diff
@@ -63,459 +63,6 @@ static const struct option req_options[] = {
     {0, 0, 0, 0} /* terminal element */
 };
 
-
-#define MAX_WIDTH 80
-#ifdef NO_WOLFSSL_REQ_PRINT
-/* print serial number out
- * return WOLFSSL_SUCCESS on success
- */
-static int _wolfSSL_X509_print_serial(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
```

Comment:
Can we keep these functions in for a couple more wolfclu releases (maybe add a #warning for users to update there version of wolfSSL). This code section allows some REQ print functionality for older wolfSSL versions that do not have the wolfSSL_X509_REQ_print function.

Reply:
Agreed. I didn't re-run configure after updating to the new wolfSSL lib, or it would have picked up the macro you added `NO_WOLFSSL_REQ_PRINT`